### PR TITLE
chore(deps): disable mkl as it is broken at the moment

### LINKED ISSUE #14: Intel mkl support
As observed here:
https://github.com/zama-ai/concrete-fftw/runs/7279783529?check_suite_focus=true

mkl builds are broken currently, could be linked to name clashes with wildcard imports from the mkl tools dependency.

Consider what should be done about mkl support

### PR DESCRIPTION
**Pull request by IceTDrinker**
For now remove the mkl feature and update the CI file.

intel mkl crate seems to be left to its own fate and currently does not compile, we remove it for the time being before a more permanent solution.

closes #14

### CODE REVIEW

**Review by aPere3 — CHANGES_REQUESTED** [review]
Thank you for removing that ! 

Unfortunately, I believe that you missed some changes to be done in some files. Grepping the repo for `mkl` at least returns:
```shell
./concrete-fftw-sys/build.rs:    if cfg!(macos) && cfg!(feature = "mkl") {
./concrete-fftw-sys/build.rs:        panic!("Mkl is not supported in the macos platform.")
./concrete-fftw-sys/src/lib.rs:#[cfg(not(feature = "mkl"))]
./concrete-fftw-sys/src/lib.rs:#[cfg(not(feature = "mkl"))]
./concrete-fftw-sys/src/lib.rs:#[cfg(feature = "mkl")]
./concrete-fftw-sys/src/lib.rs:extern crate intel_mkl_src as ffi;
```

Good luck ! :smiley:

**Comment by IceTDrinker** [comment]
> Okay for that, good idea ! But we must really make sure that we take a decision on this topic in a near future then !

Agreed, issue for it is #14

**Comment by IceTDrinker** [comment]
As discussed, mkl has been removed in the end.

**Review by aPere3 — CHANGES_REQUESTED** [review]

File: concrete-fftw/LICENSE.txt [hunk 1]
```diff
@@ -5,36 +5,7 @@ License of concrete-fftw
...
-    The above copyright notice and this permission notice shall be included in all
-    copies or substantial portions of the Software.
-
-    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-    SOFTWARE.
```
[the remark this answers is not in the document]
Comment (IceTDrinker) [reply concrete-fftw/LICENSE.txt:40]: Ah yes, my bad 🤦‍♂️

**Comment by IceTDrinker** [comment]
Done !

**Review by aPere3 — APPROVED** [review]

**Pull request merged**